pandas_gbq/auth.py

```diff
@@ -12,40 +12,34 @@
 logger = logging.getLogger(__name__)
 
 
+CREDENTIALS_CACHE_DIRNAME = "pandas_gbq"
+CREDENTIALS_CACHE_FILENAME = "bigquery_credentials.dat"
 SCOPES = ["https://www.googleapis.com/auth/bigquery"]
 
 
 def get_credentials(
-    private_key=None,
-    project_id=None,
-    reauth=False,
-    auth_local_webserver=False,
-    try_credentials=None,
+    private_key=None, project_id=None, reauth=False, auth_local_webserver=False
 ):
-    if try_credentials is None:
-        try_credentials = _try_credentials
+    import pydata_google_auth
 
     if private_key:
         return get_service_account_credentials(private_key)
 
-    # Try to retrieve Application Default Credentials
-    credentials, default_project = get_application_default_credentials(
-        try_credentials, project_id=project_id
-    )
-
-    if credentials:
-        return credentials, default_project
-
-    credentials = get_user_account_credentials(
-        try_credentials,
-        project_id=project_id,
-        reauth=reauth,
+    credentials, default_project_id = pydata_google_auth.default(
+        SCOPES,
+        client_id="495642085510-k0tmvj2m941jhre2nbqka17vqpjfddtd.apps.googleusercontent.com",
+        client_secret="kOc9wMptUtxkcIFbtZCcrEAc",
```

Comment:
Is it worth adding a note / link on what these are?

Reply:
Moved into constants and added some notes on why we include these. The gist is it's so that we can identify the app as "pandas GBQ" in the OAuth 2.0 flow.